prep-css-box-model/style.css

```diff
@@ -0,0 +1,11 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+  margin-bottom: 20px;
```

Comment:
For the margin we want to space the buttons apart from each other

Reply:
Lmao I misread "Add a margin of 20px to the BUTTON" as "Add a margin of 20px to the BOTTOM"